share/revng/mass-testing-revng-meta.yml

```diff
@@ -0,0 +1,13 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+extra_columns:
+  - name: text_size
+    label: .text size
+    type: int
+
+downloads:
+  - name: trace.json.gz
+    label: Trace
+    crash_related: true
```

Comment:
Why do we need `crash_related`?

Reply:
So that the download button only appears in the `crashes` and `timeouts` page and not in the raw data table. Unless you want to have all 3 have the same set of buttons

Reply:
Didn't we say no button but just a link to the "Index of /lol" page?

Reply:
We did, but considering that we later devised the entire `meta.yml` structure it seems a waste not to leverage it to also have direct links to files of note

Reply:
Having lots of buttons is not the nicest, I'm more than happy with index of, but as you prefer.